docs/design.md

```diff
@@ -53,130 +161,33 @@ drawbacks, for example:
    correct credentials\monitored resources), and users may be reluctant to
    “pollute” their code with OpenTelemetry.
 
-To resolve the issues above, we are introducing OpenTelemetry Agent (Agent).
+To resolve the issues above, you can run OpenTelemetry Service as an Agent.
 The Agent runs as a daemon in the VM/container and can be deployed independent
 of Library. Once Agent is deployed and running, it should be able to retrieve
 spans/stats/metrics from Library, export them to other backends. We MAY also
 give Agent the ability to push configurations (e.g sampling probability) to
 Library. For those languages that cannot do stats aggregation in process, they
 should also be able to send raw measurements and have Agent do the aggregation.
 
+TODO: update the diagram below.
+
 ![agent-architecture](https://user-images.githubusercontent.com/10536136/48792454-2a69b900-eca9-11e8-96eb-c65b2b1e4e83.png)
 
-For developers/maintainers of other libraries: Agent can also be extended to
+For developers/maintainers of other libraries: Agent can also 
 accept spans/stats/metrics from other tracing/monitoring libraries, such as
-Zipkin, Prometheus, etc. This is done by adding specific recevers. See
+Zipkin, Prometheus, etc. This is done by adding specific receivers. See
 [Receivers](#receivers) for details.
 
-To support Agent, Library should have “agent exporters”, similar to the
-existing exporters to other backends. There should be 3 separate agent
-exporters for tracing/stats/metrics respectively. Agent exporters will be
-responsible for sending spans/stats/metrics and (possibly) receiving
-configuration updates from Agent.
-
-### <a name="agent-communication"></a>Communication
```

Comment:
I think this part is still relevant.

Reply:
This describes OpenCensus protocol, which is only one of the protocols that otelsvc supports. It can be moved to a separate document describing the OpenCensus protocol. I think this document should be about general design not about one specific protocol that we support.

Reply:
Maybe put it in `protocol.md`?

Reply:
I restored the OpenCensus Protocol section and added a TODO to move it elsewhere (I am not yet sure how exactly to structure the docs, we can do it in a separate PR).